packages/design-system-docs/src/pages/components/Table/_Table.docs.scss

```diff
@@ -20,6 +20,79 @@ Markup: table-complex.example.html
 Style guide: components.table.complex
 */
 
+/*
+`<Table>`
+
+`This is a beta version of the Table components. The functionality and usage is subject to change in response to the feedback from design system users.`
```

Comment:
I don't think its necessary to say its beta here, instead use the `@status` modifier. This is hidden pretty deep in our documentation and there arent any examples so dont worry.

Adding `@status Beta` should do the trick here.

Reply:
it looks like we have the following status available: `Draft` and `Ready` and the `@status Draft` needs to be added to the top of the page in order to render unless we update the `PageBlock` script to allow for @status to be used in page sections.